dhis-2/dhis-api/src/main/java/org/hisp/dhis/security/twofa/TwoFactorType.java

```diff
@@ -32,10 +32,10 @@
 @Getter
 public enum TwoFactorType {
   NONE,
-  TOTP,
-  EMAIL,
-  ENROLLING_TOTP,
-  ENROLLING_EMAIL;
+  TOTP, // Enabled state, using TOTP
+  EMAIL, // Enabled state, using email
+  ENROLLING_TOTP, // Enrolling state, using TOTP
+  ENROLLING_EMAIL; // Enrolling state, using email
```

Comment:
if these are states then it might be better to have that represented more clearly  in the type name and states, without needing the comments. Suggestion:
```java
enum TwoFactorState {
  NOT_ENABLED,
  EMAIL_ENABLED,
  TOTP_ENABLED,
  EMAIL_ENROLLED,
  TOTP_ENROLLED;
}
```

Reply:
I like that idea.